src/contract.jl

```diff
@@ -36,3 +40,66 @@ function contract_density_matrix(
   end
   return out
 end
+
+function ITensors.scalar(
+  alg::Algorithm, tn::Union{AbstractITensorNetwork,Vector{ITensor}}; kwargs...
+)
+  return contract(alg, tn; kwargs...)[]
+end
+
+function ITensors.scalar(
+  tn::Union{AbstractITensorNetwork,Vector{ITensor}}; alg="exact", kwargs...
+)
+  return scalar(Algorithm(alg), tn; kwargs...)
+end
+
+function logscalar(
+  tn::Union{AbstractITensorNetwork,Vector{ITensor}}; alg::String="exact", kwargs...
```

Comment:
```suggestion
  tn::Union{AbstractITensorNetwork,Vector{ITensor}}; alg="exact", kwargs...
```
We shouldn't constrain to `String` since a user could input an `Algorithm` directly.